main.go

```diff
@@ -44,6 +45,16 @@ func main() {
 	logger.Infof("uploadable config: %+v", config.UploadableConfig)
 	logger.Infof("log config: %+v", config.LogConfig)
 
+	if config.Files == "" {
+		logger.Debug("No glob filter provided!")
+	} else {
+		files, err := filepath.Glob(config.Files)
+		if err != nil {
+			log.Fatalln(err)
```

Comment:
In the end, it's a validation of the glob filter. Can we find a better place then? What about validation method?